src/main/java/io/vertx/ext/mongo/AggregateOptions.java

```diff
@@ -15,7 +15,7 @@
   /**
    * The default value of batchSize = 10.
    */
-  public static final int DEFAULT_BATCH_SIZE = 20;
+  public static final int DEFAULT_BATCH_SIZE = 10;
```

Comment:
It's safer to update the docs rather than the code: such a change my impact behavior of applications running in production.

Reply:
Thanks for that, only fixed it in FindOptions